src/fragmentarium/ui/search/ChapterInfoResults.tsx

```diff
@@ -134,24 +179,13 @@ function TransliterationSearchPagination({
         </Col>
       </Row>
       <Row>
-        <Col>
-          <Table responsive>
-            <thead>
-              <tr>
-                <th>Text</th>
-                <th>Chapter</th>
-                <th>Matching Lines</th>
-              </tr>
-            </thead>
-            <tbody>{PaginationElementComponent}</tbody>
-          </Table>
-        </Col>
+        <Col>{PaginationElementComponent}</Col>
       </Row>
       <Row></Row>
```

Comment:
Do you know what the empty row does here? (I know it's not related to the changes in the PR).

Reply:
these rows are not needed